esx_service/vmdk_ops.py

```diff
@@ -668,20 +670,27 @@ def vm_uuid2name(vm_uuid):
         return None
     return vm.config.name
 
-# Return error, or None for OK.
-def attachVMDK(vmdk_path, vm_uuid):
-    vm = findVmByUuid(vm_uuid)
-    logging.info("*** attachVMDK: %s to %s VM uuid = %s",
-                 vmdk_path, vm.config.name, vm_uuid)
-    return disk_attach(vmdk_path, vm)
 
+def attach_or_detach_VMDK(cmd, vmdk_path, vm_name, bios_uuid, vc_uuid):
```

Comment:
Yes, I understand what you had in mind, it's just my taste preferences are different :-)
SO I guess my fix is a reasonable compromise, I will stick with it.